Enterprise-CMCS/cmcs-eregulations
# EREGCSC-1774 -- Remove prototype

Resolves [EREGCSC-1774](https://jiraent.cms.gov/browse/EREGCSC-1774)

**Description**

The prototype site is no longer needed.  Its code was being kept around, undeleted, because we (I) thought that it might be a good source of inspiration or guidance going forward.

Instead, I've never looked at it once and the current state of the application (in both content and coding style) has made anything in the prototype folder obsolete and unnecessary.  

And further: our automated code quality tools are crawling this obsolete, vestigial directory and creating broken, superfluous pull requests that target this code.  

It's time to say goodbye to the prototype.  Thanks for all the fish.

**This pull request changes:**

- DELETES prototype directory and most of its contents, including all configuration files, dependencies, and related documentation
- KEEPS all Vue Components and Storybook Story files related to the Synthetic part work.  See [PR](https://github.com/Enterprise-CMCS/cmcs-eregulations/pull/536) and [JIRA story for EREGCSC-1482](https://jiraent.cms.gov/browse/EREGCSC-1482) for context.

> **Note**
> Our Storybook implementation has been partially broken for quite some time, so even though we're keeping the Synthetic Part components (see above), they are not currently viewable using Storybook.  They weren't working before this story began, either.  So if they are to be viewed in Storybook again, work will need to be done to make that happen.

**Steps to manually verify this change:**

1. Search directory for evidence of the prototype and hopefully not find anything
2. Experimental Deployment works as expected and a quick smoke test of the site shows no stylistic regressions
3. Snyk PRs either close themselves once all the configuration/dependency code is removed, or we close the PRs and hope they aren't automatically reopened
